roles/sriov_config/README.md

```diff
@@ -73,6 +73,7 @@ The `sriov_config_file` **must** meet the following requirements
 | meta_plugins      | No       | String  | MetaPluginsConfig configuration to be used in order to chain metaplugins to the sriov interface returned by the operator.
 | min_tx_rate       | No       | Int     | Minimum tx rate, in Mbps, for the VF, min_tx_rate should be <= max_tx_rate.
 | network_namespace | No       | String  | Namespace of the NetworkAttachmentDefinition custom resource
+| plugins           | No       | String  | Plugins to be configured for this network.
```

Comment:
`plugins` is not a field supported by [SRIOV Network.](https://github.com/openshift/sriov-network-operator/blob/master/manifests/stable/sriovnetwork.openshift.io_sriovnetworks.yaml) Are you sure this is where you want to add that setting?